ketcher-autotests/tests/User-Interface/Floating-windows/floating-windows.spec.ts

```diff
@@ -0,0 +1,156 @@
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+    await page
+      .getByRole('listitem')
+      .filter({ hasText: 'Molecular Weight:Decimal places3' })
+      .getByRole('button', { name: '3' })
+      .click();
+    await page.getByRole('option', { name: '0' }).click();
+    await page.getByRole('button', { name: '3', exact: true }).click();
+    await page.getByRole('option', { name: '0' }).click();
+  });
+
+  test('Calculated values: check accuracy 2', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-3999(2)
+      Description: verify 7 decimal places after the dot for calculated values 
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+    await page
+      .getByRole('listitem')
+      .filter({ hasText: 'Molecular Weight:Decimal places3' })
+      .getByRole('button', { name: '3' })
+      .click();
+    await page.getByRole('option', { name: '7' }).click();
+    await page.getByRole('button', { name: '3', exact: true }).click();
+    await page.getByRole('option', { name: '7' }).click();
+  });
+
+  test('Calculate values: verify UI (empty canvas)', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-4002
+      Description: verify empty fields in floating window for empty canvas 
+    */
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+  });
+
+  test('Open structure: Open window', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-4003
+      Description: verify floating window for 
+      open/drag file or paste from clipboard 
+    */
+    await selectTopPanelButton(TopPanelButton.Open, page);
+  });
+
+  test('Floating windows - Extended table: Verify UI', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-4010
+      Description: verify visual representation of "Extended" table 
+    */
+    await page.getByTestId('extended-table').click();
+  });
+
+  test('Calculated Values', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-4000
+      Description: Change dedcimal places
+    */
+    await openFileAndAddToCanvas('KET/calculated-values-chain.ket', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+    await page.getByText('Decimal places3').first().click();
+    await page.getByRole('option', { name: '4' }).click();
+    await page.getByText('Decimal places3').click();
+    await page.getByRole('option', { name: '1' }).click();
+    await page.keyboard.press('Escape');
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+  });
+
+  test('Opening text file', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-4005, EPMLSOPKET-4009
+      Description: open text file via "open file" 
+    */
+    await selectTopPanelButton(TopPanelButton.Open, page);
+    await openFile('cml-1945.cml', page);
```

Comment:
Could you, please, rename file and move to corresponding folder?

Reply:
Done.